include/packager/live_packager.h

```diff
@@ -125,6 +125,36 @@ class LivePackager {
   LiveConfig config_;
 };
 
+struct PSSHData {
+  std::vector<uint8_t> cenc_box;
+  std::vector<uint8_t> mspr_box;
+  std::vector<uint8_t> mspr_pro;
+  std::vector<uint8_t> wv_box;
+};
+
+enum struct EncryptionSchemeFourCC : uint32_t {
+  CBCS = 0x63626373,
```

Comment:
We already have `EncryptionScheme` defined in `LiveConfig`.  Can we somehow unite them together?

Reply:
How?

Reply:
We can use this approach: https://github.com/Pluto-tv/service-ingest-media/blob/main/internal/model/encryption.go#L24

Reply:
After some discussions with Vadym, it was decided to rename it and leave it as is.

As I mentioned previously, merging this enum with EncryptionScheme would introduce the possibility to pass incorrect parameters just by picking wrong value from the enum

It is also a stand-alone functionality we would not want to mix with LivePackager

Reply:
Since this is unique to PSSH generation, let's move the enum inside of `PSSHGeneratorInput` so there's no confusion.